bam/src/configuration/applier/kpi.cc

```diff
@@ -185,8 +187,10 @@ void applier::kpi::apply(
   //
   for (std::map<unsigned int, applied>::const_iterator
          kpi_it(_applied.begin()),
+         next_kpi_it(_applied.begin()),
```

Comment:
Again, neither __resolve_kpi()_ nor __invalidate_ba()_ seem to modify __applied_. So _it_ iterator should be valid, right ?